activerecord/test/cases/associations/has_and_belongs_to_many_associations_test.rb

```diff
@@ -824,18 +824,18 @@ def test_association_proxy_transaction_method_starts_transaction_in_association_
   end
 
   def test_caching_of_columns
-    david = Developer.find(1)
+    Developer.find(1)
```

Comment:
It would be funny how someone would theories  why this is needed.

```suggestion
```

Reply:
The whole test is weird to be honest, and has no place in this file after the refactor. I'll have to do some git blame to figure what the intent was.

Reply:
Yeah, comes from 16 years ago: https://github.com/rails/rails/commit/3ee4e009185173aab78f6503ee45e3ef4482874e, back in those days there wasn't a central schema cache, so associations where holding columns lists.

This test is no entirely irrelevant, I'll just 🔥 it.